OWNERS

```diff
@@ -1,27 +1,20 @@
 approvers:
 - justinsb
-- zmerlynn
-- gnufied
```

Comment:
cc @gnufied https://github.com/kubernetes/cloud-provider-aws/commits?author=gnufied

Reply:
I would still preferably keep myself here. I am primary maintainer/author/reviewer of several features and bug fixes in AWS's storage layer. I don't usually review other cloudprovider features though.

Reply:
None of the storage integration are included in the CCM component, are you still interested in reviewing PRs here?

Reply:
Worth noting that the core functionality in the CCM is still imported from https://github.com/kubernetes/legacy-cloud-providers/tree/master/aws which you are still an approver for.

Reply:
Ack